export/distro/registrations.go

```diff
@@ -158,6 +158,17 @@ func (reg registrationInfo) processEvent(event *models.Event) {
 	}
 
 	reg.sender.Send(encrypted, event)
+
+	if configuration.MarkPushed {
+		go func(id string) {
```

Comment:
Why not utilize a goroutine on line 183 of registrations.go instead?
```
case event := <-reg.chEvent:
reg.processEvent(event)
```
Pass event in as a variable to an anonymous function invoked via goroutine....?